src/components/molecules/Visualizer/Widget/index.tsx

```diff
@@ -71,6 +72,12 @@ export default function WidgetComponent<PP = any, SP = any>({
     [widget.id, onExtend],
   );
 
+  const iFrameProps = useMemo(() => {
+    return {
+      style: { pointerEvents: (editing ? "none" : "auto") as any },
```

Comment:
Is there a reason you are type casting as any?

Reply:
This is only because i did not find the defination of PointerEvents and use string will cause an type error. I've update this part and also memo the style for Builtin component.